javascript/code-challenges/challenges-13.test.js

```diff
@@ -0,0 +1,430 @@
+"use strict";
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function named longestString that takes in an array of strings and returns the index position of the longest string.
+------------------------------------------------------------------------------------------------ */
+
+const longestString = (arr) => {
+  // if (!arr.length) {
+  //   return -1;
+  // }
+
+  // let idx = 0;
+  // let longestLength = 0;
+
+  // for (let i = 0; i < arr.length; i++) {
+  //   if (arr[i].length > longestLength) {
+  //     idx = i;
+  //     longestLength = arr[i].length;
+  //   }
+  // }
+
+  // return idx;
```

Comment:
Great work on this challenge! 

Feedback: Try your best to catch old code/commented out code before merge. Always good to take a look through files and do a quick clean up 👍